packages/quick_actions/quick_actions_ios/example/ios/RunnerTests/QuickActionsPluginTests.swift

```diff
@@ -7,47 +7,43 @@ import XCTest
 
 @testable import quick_actions_ios
 
+class MockFlutterApi: IOSQuickActionsFlutterApiProtocol {
+  /// Method to allow for async testing.
+  var testStub: ((String) -> Void)? = nil
```

Comment:
How about `launchActionCallback` or something similar, so it's more descriptive?